### PR TITLE
Merge ip6 with history into epic repository

### PR DESCRIPTION
**Pull request by wdconinc**
### Briefly, what does this PR introduce?
Merge ip6 master branch into epic.

### What kind of change does this PR introduce?
- [X] Bug fix (issue #271)
- [ ] New feature (issue #__)
- [ ] Documentation update
- [ ] Other: __

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [ ] Documentation has been added / updated
- [ ] Changes have been communicated to collaborators

### Does this PR introduce breaking changes? What changes might users need to make to their code?
Some benchmarks will need updates.

### Does this PR change default behavior?
Yes, ip6 now lives inside the epic repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,17 +14,10 @@ The EPIC Detector at IP6 for Electron-Ion Collider experiment.
...
     - [ToF](https://eic.phy.anl.gov/geoviewer/index.htm?nobrowser&file=https://eicweb.phy.anl.gov/EIC/detectors/ecce/-/jobs/artifacts/main/raw/geo/detector_geo_tof_only.root?job=dump_geometry&item=default;1&opt=clipx;clipy;transp30;zoom55;ROTY49;ROTZ350;trz0;trr0;ctrl;all&)
   - [Tracking](https://eic.phy.anl.gov/geoviewer/index.htm?nobrowser&file=https://eicweb.phy.anl.gov/EIC/detectors/ecce/-/jobs/artifacts/main/raw/geo/detector_geo_tracking_only.root?job=dump_geometry&item=default;1&opt=clipx;clipy;transp30;zoom75;ROTY320;ROTZ340;trz0;trr0;ctrl;all)
-    - [Vertex detector](https://eic.phy.anl.gov/geoviewer/index.htm?nobrowser&file=https://eicweb.phy.anl.gov/EIC/detectors/ecce/-/jobs/artifacts/main/raw/geo/vertex_only_geo.root?job=dump_geometry&item=default;1&opt=clipx;clipy;transp30;zoom120;ROTY320;ROTZ340;trz0;trr0;ctrl;all)
+    - [Vertex detector](https://eic.phy.anl.gov/geoviewer/index.htm?nobrowser&file=https://eicweb.phy.anl.gov/EIC/detectors/ecce/-/jobs/artifacts/maialtaltn/raw/geo/vertex_only_geo.root?job=dump_geometry&item=default;1&opt=clipx;clipy;transp30;zoom120;ROTY320;ROTZ340;trz0;trr0;ctrl;all)
```
Comment (veprbl) [inline README.md:17]:
```suggestion
    - [Vertex detector](https://eic.phy.anl.gov/geoviewer/index.htm?nobrowser&file=https://eicweb.phy.anl.gov/EIC/detectors/ecce/-/jobs/artifacts/main/raw/geo/vertex_only_geo.root?job=dump_geometry&item=default;1&opt=clipx;clipy;transp30;zoom120;ROTY320;ROTZ340;trz0;trr0;ctrl;all)
```

**Review by veprbl — APPROVED** [review]
Seems to work!

**Review by veprbl — APPROVED** [review]

**Comment by veprbl** [comment]
@wdconinc This needs a merge instead of squash

**Comment by wdconinc** [comment]
Yes, indeed. I'll temporarily change the settings.

**Review by ajentsch — APPROVED** [review]

**Pull request merged**